### DISCUSSION TITLE
GraphQL Indexer / Dictionary on Ternoa - Learn from examples

### DISCUSSION THREAD

**Discussion by Sha0-o-0** [capsule-corp-ternoa/ternoa-js#179] [Show and tell]
Hello Ternoa Builders,

I've created simple "Learn from examples" tutorials for everyone looking to learn how to create custom GraphQL requests using Ternoa Indexer and Dictionary.

Each file covers different elements with a step by step method of learning with examples.

1. Indexer 101 : Filters
2. Indexer 102 : Aggregates
3. Indexer 103 : SubQueries

Process
- open https://indexer-mainnet.ternoa.dev/
- copy paste the file content
- read comments and run queries in order
- try to make your own queries

[indexer_101_filters.txt](https://github.com/capsule-corp-ternoa/ternoa-js/files/12580764/indexer_101_filters.txt)
[indexer_102_aggregates.txt](https://github.com/capsule-corp-ternoa/ternoa-js/files/12580765/indexer_102_aggregates.txt)
[indexer_103_subQueries.txt](https://github.com/capsule-corp-ternoa/ternoa-js/files/12580766/indexer_103_subQueries.txt)

Let's build !

Sha0

**Comment by Victor-Salomon** [comment]
@Sha0-o-0 amazing workshop tutorial!! 
I will see how to integrate a link to this page on the official Ternoa Documentation if it's good for you. 

I still have one request: This learning thread is on the SDK repository, while the topic of this tutorial only concerns the indexer. 
Do you mind creating this discussion [on the indexer repository](https://github.com/capsule-corp-ternoa/ternoa-subql/discussions)? 

Thank you again for your contribution.  
It's much appreciated! 

Let's build! 😉

**Reply by Sha0-o-0** [reply]
Hey @Victor-Salomon 

Thanks for feedback ! Hope it can help the community ;)

Just created a new discussion here https://github.com/capsule-corp-ternoa/ternoa-subql/discussions/185
